quadPlot.py

```diff
@@ -15,15 +15,18 @@
 history = np.zeros((500,3))
 count = 0
 
-def plot_quad_3d(waypoints, args=()):
+def plot_quad_3d(waypoints, init_pos, args=()):
```

Comment:
see if you can adapt your change to my thread-less part, I will review this part later on.  I suggest you seperate this review into two, one is general clean up and quadPlot related change, the other is lqr controller. You can do this in your branch now and submit incremental change, which it is easy for me to review.